src/main/java/com/lorepo/icplayer/client/module/choice/OptionView.java

```diff
@@ -73,11 +72,11 @@ private native void setListener(Element el)/*-{
 		var $el = $wnd.$(el);
 		var self = this;
 		$el.on('touchend',function(e){ //onBrowserEvent is not used to avoid visible delay
-			var isFromInsertedAddon = $wnd.$(e.target).is("input");
-			if (isFromInsertedAddon) return;
+			var isFromInsertedAudio = $wnd.$(e.target).is("input");
+			if (isFromInsertedAudio) return;
 			
-			var isFromAddedAddon = $wnd.$(e.target).parents().hasClass("inner_addon");
-			if (isFromAddedAddon) return;
+			var isFromAddedAudio = $wnd.$(e.target).parents().hasClass("inner_addon");
```

Comment:
tutaj już addon mogło zostać, ale dobra, technicznie rzecz biorąc jest prawidłowo bo ten fragment ma przede wszystkim obsłużyć addon Audio. Jeśli wróci, to bym przywrócił tu addon, jeśli nie, to może zostać.